floss/results.py

```diff
@@ -139,7 +139,7 @@ def from_utf8(cls, buf, addr, min_length):
         except UnicodeDecodeError:
             raise ValueError("not utf-8")
 
-        if not decoded_string.isprintable():
+        if not decoded_string.replace("\n", "").isprintable():
```

Comment:
from the docs: "Nonprintable characters are those characters defined in the Unicode character database as “Other” or “Separator”, excepting the ASCII space (0x20) which is considered printable."

I think we should use a different check here or remove as we've already proven this UTF-8 decodes successfully?

Reply:
Agreed, let's remove the check.😄